include/fizzy/fizzy.h

```diff
@@ -133,6 +133,33 @@ typedef struct FizzyExternalGlobal
     FizzyGlobalType type;
 } FizzyExternalGlobal;
 
+/// External kind.
+typedef enum FizzyExternalKind
+{
+    FizzyExternalKindFunction,
+    FizzyExternalKindTable,
+    FizzyExternalKindMemory,
+    FizzyExternalKindGlobal
+} FizzyExternalKind;
+
+/// Import description.
+///
+/// @note Only one member of FizzyImportDescription::desc union corresponding to
+/// FizzyImportDescription::kind is defined for each import.
+typedef struct FizzyImportDescription
+{
+    const char* module;
```

Comment:
These fields are documented in #686. It should be consistent, either document here too, or not in #686.